src/Components/Calculator.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor() {
+    super();
+    this.state = {
+      a: 0,
+      b: 0,
+      operation: '',
+      displayed: 0,
+    }
+  }
+
+  clear = () => {
```

Comment:
1) Please do not use arrow functions for class methods.
2) You don't use this function anywhere, maybe because some PRs weren't merged. After they get merged, please use this action. Right now we cannot say that the task is fully implemented.